### ISSUE TITLE
Proposal to obsolete 'GO:0019015 viral genome' and its children

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#16993]
Hello, 

The proposal has been made to obsolete 
GO:0019015 viral genome (3 annotations)  its children: 
GO:0019021 DNA viral genome 		-> 	0 annotations
GO:0019016 non-segmented viral genome -> 	0 annotations
GO:0019038 provirus 		-> 	146 annotations
GO:0019022 RNA viral genome -> 	0 annotations
GO:0019023 dsRNA viral genome -> 	0 annotations
GO:0019024 ssRNA viral genome -> 	0 annotations
GO:0019017 segmented viral genome -> 	0 annotations 
GO:0019018 bipartite viral genome -> 	0 annotations 
GO:0019020 multipartite viral genome -> 	0 annotations 
GO:0019019 tripartite viral genome  -> 	0 annotations 

The reason for obsoletion is that these terms do not represent GO cellular components. 

Most annotations are from JCVI and TIGR except 2 ISS from EcoCyc, that are missing the 'with' information. 

There are no mappings to these terms; these terms are not parts of any subsets (slims). 

Any comments can be  added to the issue: https://github.com/geneontology/go-ontology/issues/16993

We are opening a comment period for this proposed obsoletion. We’d like to proceed and obsolete this term on March 12th, 2019. Unless objections are received by March 12th, 2019. we will assume that you agree to this change.

Thanks, Pascale

**Comment by keseler** [comment]
IMO GO:0019038 provirus should not be obsoleted; it does have the wrong parentage, though.  It probably should be a child of GO:0098687 chromosomal region.

Cheers,
Ingrid

**Comment by pgaudet** [comment]
@mgiglio99 You had proposed the obsoletion of provirus - I just wanted to know if you agree with the proposal to move it under 'chromosomal region' instead ? 

Thanks, Pascale

**Comment by mgiglio99** [comment]
It is a chromosomal region so I guess that could work. 
Still not sure how it would be used in annotation.

**Comment by pgaudet** [comment]
@keseler Any candidate proteins for annotation to 'provirus'? 

Note that we have a few processes related to proviruses: 

- 'provirus excision'
- 'establishment of integrated proviral latency' 
- 'establishment of viral latency' 
- 'viral genome integration into host DNA' 

Thanks, Pascale

**Comment by keseler** [comment]
I had to really think about that... ;-)  (For so long, in fact, that your comment appeared in the meantime!) I suspect that previous annotations may have been using the provirus term in a kind of SO sense, not in the proper "localizes to this cellular component" sense.  One of the EcoCyc proteins currently annotated with the term, AlpA, seems to be a transcriptional regulator of expression of the phage integrase, so I could envision it having a "localizes to" annotation to provirus.  On the other hand, it is tempting confusion...

In other words, I realize that there are better process/function GO terms that could be used for annotation, and wouldn't be devastated if you decided to obsolete the CC provirus term.

Cheers,
Ingrid

**Comment by pgaudet** [comment]
That's interesting ! I don't even find a CC term that corresponds to a promoter - we only have MF terms, with which we plan to use a SO term to define the location of the DNA binding. 

I'll obsolete it too, to avoid confusion. 

Thanks, Pascale
